packages/e2e/features/ui/components/authenticator/sign-in-federated.feature

```diff
@@ -9,14 +9,14 @@ Feature: Sign In with Federation
   @angular @react @vue
   Scenario: Sign In screen has social providers
     Then I see "Sign In with Amazon"
-    And I see "Sign In with Apple"
+    # And I see "Sign In with Apple"
```

Comment:
We don't have apple set up yet, because that requires a monthly subscription fee. We can definitely do this, but skipping apple for now.